teachertool/src/transforms/loadProjectMetadataAsync.ts

```diff
@@ -34,5 +34,5 @@ export async function loadProjectMetadataAsync(inputText: string, shareLink: str
     };
     dispatch(Actions.setProjectMetadata(projectData));
     initNewProjectResults();
-    logDebug(`Loaded project metadata: ${JSON.stringify(projMeta)}`);
+    logDebug("Loaded project metadata", projMeta);
```

Comment:
Why is the `JSON.stringify` no longer applied to the `projMeta`?

Reply:
It's now passed into the log as an object, which allows the console to display it in a more user-friendly fashion (you can expand/collapse fields instead of having just one long string)

Reply:
Like this:
![image](https://github.com/microsoft/pxt/assets/69657545/95b204da-c95d-4fbe-aa1c-3bba56fb69d9)

Reply:
Oh, cool! Thanks for clarifying!